lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -73,7 +73,10 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     _addFontSizeObserver();
     _addLocaleChangedListener();
     registerHotRestartListener(dispose);
-    _setAppLifecycleState(ui.AppLifecycleState.resumed);
+    appLifecycleStateSupport = DomAppLifecycleStateSupport(
+      broadcastLifecycleState: _setAppLifecycleState,
+    );
+    _setAppLifecycleState(appLifecycleStateSupport.appLifecycleState);
```

Comment:
Minor suggestion: broadcast this initial state from the constructor of the `DomAppLifecycleStateSupport`, so we don't need to manually call the setter here.

(If we ever want to prevent this behavior, we can add a `broadcastInitialState` or similar to the constructor to enable/disable it, but for now, it seems that we always want to broadcast the `appLifecycleState` next to the creation of the DomAppLifecycleStateSupport object, so I'd move it to the constructor).